mycroft/messagebus/service/main.py

```diff
@@ -14,16 +14,13 @@
 #
 # You should have received a copy of the GNU General Public License
 # along with Mycroft Core.  If not, see <http://www.gnu.org/licenses/>.
-
 import tornado.ioloop as ioloop
 import tornado.web as web
-import tornado.autoreload as autoreload
```

Comment:
This is missing and hinders the messagebus service from starting

Reply:
Good catch! It's strange that despite the alias, it still imports the original name. Either way, I've changed it to use the alias now (and changed `import mod.mod as mod` to `from mod import mod`).